package.json

```diff
@@ -35,14 +35,14 @@
   "homepage": "https://github.com/vacuumlabs/cardano-hw-cli#readme",
   "dependencies": {
     "@babel/runtime": "^7.11.2",
-    "@cardano-foundation/ledgerjs-hw-app-cardano": "https://github.com/vacuumlabs/ledgerjs-cardano-shelley/releases/download/v2.0.2-rc.5/cardano-foundation-ledgerjs-hw-app-cardano-v2.0.2-rc.5.tgz",
+    "@cardano-foundation/ledgerjs-hw-app-cardano": "https://github.com/vacuumlabs/ledgerjs-cardano-shelley/releases/download/v2.2.0-beta/cardano-foundation-ledgerjs-hw-app-cardano-2.2.0-beta.tgz",
```

Comment:
you can already switch to npm release of ledgerjs

Reply:
Done

Reply:
This seems like we are still using the github prerelease